### PR TITLE
Update CONTRIBUTING

### PR DESCRIPTION
**Pull request by dcousens**
Resolves https://github.com/JedWatson/react-select/issues/5464

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ Before you open a PR:
   know. We'll help you get started, rather than inadvertently doubling up on your hard work.
 - Make sure you do not add regressions by running `yarn test`.
 - Where possible, include tests with your changes, either that demonstrates the
-  bug, or tests the new functionality. If you're not sure how to test your
-  changes, feel free to ping @gwyneplaine
-- Please [follow our established coding conventions](https://github.com/keystonejs/keystone/wiki/Coding-Standards)
+  bug, or tests the new functionality.
+- Please [follow our contributors guidelines]([https://github.com/keystonejs/keystone/wiki/Coding-Standard](https://github.com/JedWatson/react-select/blob/master/CONTRIBUTING.md)).
```
Comment (Rall3n) [inline CONTRIBUTING.md:22]: Markdown link has incorrect format and does not work, and link points to the same file it originates on (`CONTRIBUTING.md`).

**Review by Rall3n — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (dcousens) [reply CONTRIBUTING.md]: Sorry, bad copy paste

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,8 +18,6 @@ Before you open a PR:
   know. We'll help you get started, rather than inadvertently doubling up on your hard work.
 - Make sure you do not add regressions by running `yarn test`.
 - Where possible, include tests with your changes, either that demonstrates the
-  bug, or tests the new functionality. If you're not sure how to test your
-  changes, feel free to ping @gwyneplaine
-- Please [follow our established coding conventions](https://github.com/keystonejs/keystone/wiki/Coding-Standards)
-  (with regards to formatting, etc)
+  bug, or tests the new functionality.
+- Please [follow our contributors guidelines](https://github.com/JedWatson/react-select/blob/master/CONTRIBUTING.md).
```
Comment (lukebennett88) [inline CONTRIBUTING.md:22]: Is there any point this being a link if it just links to the current document?

Comment (dcousens) [reply CONTRIBUTING.md:22]: No, let's remove the line completely

**Review by lukebennett88 — APPROVED** [review]

**Pull request merged**